test_util/src/servers/registry.rs

```diff
@@ -59,6 +61,16 @@ async fn registry_server_handler(
     return Ok(res);
   }
 
+  // serve the registry package files
+  let mut file_path =
+    testdata_path().to_path_buf().join("jsr").join("registry");
+  file_path.push(&req.uri().path()[1..].replace("%2f", "/"));
+  if let Ok(body) = tokio::fs::read(&file_path).await {
+    return Ok(Response::new(UnsyncBoxBody::new(
+      http_body_util::Full::new(Bytes::from(body)),
+    )));
+  }
```

Comment:
This confused for a few minutes, but we have ended up with two JSR test registries. I'll circle back with another PR that gets all the JSR tests to use this "test registry server".